pkg/common/utils/utils.go

```diff
@@ -240,18 +240,21 @@ func MergeMaps(first, second map[string]string) map[string]string {
 // find user name from pod label
 func GetUserFromPod(pod *v1.Pod) string {
 	userLabelKey := conf.GetSchedulerConf().UserLabelKey
-	// User name to be defined in labels
-	for name, value := range pod.Labels {
-		if name == userLabelKey {
-			log.Logger().Info("Found user name from pod labels.",
-				zap.String("userLabel", userLabelKey), zap.String("user", value))
-			return value
-		}
+	// UserLabelKey should not be empty
+	if len(userLabelKey) == 0 {
+		userLabelKey = constants.DefaultUserLabel
+		log.Logger().Debug("userLabelKey is empty and replaced by default setting",
+			zap.String("userLabel", userLabelKey))
```

Comment:
The default is documented and we should not need to print it every time we try to retrieve a user, simple like this: "userLabelKey not set using default value"

Need to also handle this as part of a unit test, codecov complains now

Reply:
Hi @wilfred-s, I had added the relevant thing to the latest commit.
The empty userLabelKey case was added to the unit test.
The log info was removed